### ISSUE TITLE
Error when run inference_example.sh 

### ISSUE THREAD

**Issue by peterwilson3912** [yewzijian/3DFeatNet#3]
When I run `inference_example.sh`, I faced the following error. I have create a new folder called ckpt.
```
 File "3DFeatNet-master/venv/lib/python3.5/site-packages/tensorflow/python/framework/ops.py", line 1470, in __init__
    self._traceback = self._graph._extract_stack()  # pylint: disable=protected-access

NotFoundError (see above for traceback): Unsuccessful TensorSliceReader constructor: Failed to find any matching files for ./ckpt/checkpoint.ckpt
         [[Node: save/RestoreV2_3 = RestoreV2[dtypes=[DT_FLOAT], _device="/job:localhost/replica:0/task:0/device:CPU:0"](_arg_save/Const_0_0, save/RestoreV2_3/tensor_names, save/RestoreV2_3/shape_and_slices)]]
         [[Node: save/RestoreV2_24/_53 = _Recv[client_terminated=false, recv_device="/job:localhost/replica:0/task:0/device:GPU:0", send_device="/job:localhost/replica:0/task:0/device:CPU:0", send_device_incarnation=1, tensor_name="edge_172_save/RestoreV2_24", tensor_type=DT_FLOAT, _device="/job:localhost/replica:0/task:0/device:GPU:0"]()]]
```
The training step works well. By the way, what loss is recommanded to stop training?
Thank you so much for your help.

**Comment by yewzijian** [comment]
Hi, you need to place the training checkpoint in the ckpt folder. you can use the ones from your training or the one I provided (the link is in the readme). 

For training, you can just let it run all the way as in my train.sh (which runs for 2+70 epochs). That should be sufficient for the model to converge.
